terraform/shared/shared.tf.json.template.py

```diff
@@ -403,28 +406,34 @@ def paren(s: str) -> str:
                         })
                     }
                 },
-                'aws_cloudwatch_event_bus': {
-                    'securityhub': {
-                        'name': 'securityhub'
-                    }
-                },
                 'aws_cloudwatch_event_rule': {
                     'inspector': {
                         'name': 'inspector',
-                        'event_bus_name': '${aws_cloudwatch_event_bus.securityhub.arn}',
                         'event_pattern': json.dumps({
-                            'source': ['aws.securityhub'],
-                            'detail-type': ['Security Hub Findings - Imported'],
-                            'detail.ProductArn': ['arn:aws:securityhub:us-east-1::product/aws/inspector'],
-                            'detail.Severity.Label': ['CRITICAL', 'HIGH']
+                            'source': ['aws.inspector2'],
+                            'detail-type': ['Inspector2 Finding'],
+                            'detail.severity': ['CRITICAL', 'HIGH'],
+                            'detail.status': ['ACTIVE']
                         })
                     }
                 },
                 'aws_cloudwatch_event_target': {
                     'inspector_to_sns': {
-                        'event_bus_name': '${aws_cloudwatch_event_bus.securityhub.arn}',
                         'rule': '${aws_cloudwatch_event_rule.inspector.name}',
-                        'arn': '${aws_sns_topic.monitoring.arn}'
+                        'arn': '${aws_sns_topic.monitoring.arn}',
+                        'input_transformer': {
+                            # Note that the input template should be valid JSON
+                            # _after_ AWS EventBridge (at runtime) replaces the
+                            # variables wrapped in `<` and `>` with the
+                            # corresponding values. Pre-defined variables that
+                            # refer to JSON objects cannot be placed in quotes.
+                            # https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-transform-target-input.html
+                            'input_template': dedent(f'''\
+                            {{
+                                "deployment": "{config.deployment_stage}",
+                                "event": <aws.events.event.json>
+                            }}''')
```

Comment:
```suggestion
                            'input_template': json.dumps({
                                'deployment': config.deployment_stage,
                                'event': {}
                            }).replace('{}','<aws.events.event.json>')
```

Avoids the backslash, the reintroduction of `dedent`, and the need to double-up the curlies.